.github/workflows/build_and_test_dbt.yaml

```diff
@@ -6,6 +6,16 @@ on:
   push:
     branches: [master]
 
+  workflow_dispatch:
+    inputs:
+      models:
+        required: true
```

Comment:
I _think_ this will prevent an empty string from being passed (the docs are sparse on this). Going to let dbt handle any errors from passing malformed model names.

Reply:
**[Thought, non-blocking]** I think requiring this variable and letting dbt Core handle validation makes sense! An alternative interface we could consider would be to allow this variable to be empty and to set a `default` of `path:models`, which will build everything. But building everything is currently a pretty intensive operation, so I also think it would make sense to keep the current design and require users to ask for that explicitly.